.yarn/patches/linkedom-npm-0.15.3-869d20f04f.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/package.json b/package.json
+index 9530332458a922de95d94d13f69c83b9ac1e927f..05ca826e3bf5d5075bc5b463823c2ef6e8b08112 100644
+--- a/package.json
++++ b/package.json
+@@ -49,6 +49,7 @@
+       "default": "./cjs/index.js"
+     },
+     "./cached": {
++      "types": "./types/esm/index.d.ts",
```

Comment:
if there is a problem, this wont help with user installs, right? it will only affect our own install

Reply:
Corect, it's a patch that's only installed when you develop in crawlee

Reply:
so will it work fine for our users or not? :]